pkg/verifier/notation/certstoresbytype.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright The Ratify Authors.
```

Comment:
does this file need a test?

Reply:
![image](https://github.com/ratify-project/ratify/assets/74894646/8eaf004b-f810-4c85-af56-60b428de5a13)